### PR TITLE
feat(LogSnippet): rename leftBorderVariant prop, remove enum

### LINKED ISSUE #668: [component-groups | LogSnippet] - Rename leftBorderVariant
Follow up to breaking change [PR](https://github.com/patternfly/react-component-groups/pull/145)

We've renamed LogSnippet prop because of design changes and switching from border variant to alert variant 

rename
`leftBorderVariant` to `variant`

The `LogSnippetBorderVariant` has been removed and replaced with PF `AlertVariant`.  Perhaps the current values could be mapped to the new ones since the types almost match

Example:
```
<LogSnippet message='Failure - check logs for details' logSnippet={code} leftBorderVariant={LogSnippetBorderVariant.success} />
```
becomes
```
<LogSnippet message='Failure - check logs for details' logSnippet={code} variant={AlertVariant.success} />
```
or
```
<LogSnippet message='Failure - check logs for details' logSnippet={code} variant="sucess" />
```

Required actions:
- Build codemod
- Build test
- Update readme with description & example

### PR DESCRIPTION
**Pull request by adamviktora**
Closes #668

### CODE REVIEW

File: packages/eslint-plugin-pf-codemods/src/rules/helpers/getFromPackage.ts [hunk 1]
```diff
@@ -103,6 +104,10 @@ export function getDefaultImportsFromPackage(
   ) as ImportDeclaration[];
 
   return importDeclarationsFromPackage
-    .filter((imp) => imp.specifiers[0]?.type === "ImportDefaultSpecifier")
+    .filter(
+      (imp) =>
+        imp.source.value?.toString().includes(componentName) &&
```
Comment (thatblindgeye) [inline packages/eslint-plugin-pf-codemods/src/rules/helpers/getFromPackage.ts:109]: Would we always want to  run this `includes` check, or only if componentName is passed a non-empty value?

**Review by thatblindgeye — COMMENTED** [review]
Just one question below, otherwise lgtm

File: packages/eslint-plugin-pf-codemods/src/rules/helpers/getFromPackage.ts (hunk 1 above)
Comment (adamviktora) [reply packages/eslint-plugin-pf-codemods/src/rules/helpers/getFromPackage.ts:109]: Yeah, that should run always. The idea was that if no value is passed, the includes check will return true. But it is an unnecessary check, let me rewrite that

File: packages/eslint-plugin-pf-codemods/src/rules/v6/componentGroups-logSnippetRenameLeftBorderVariant/componentGroups-logSnippet-rename-leftBorderVariant.ts [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+              logSnippetBorderVariantImports
+                .map((imp) => imp.local.name)
+                .includes(node.object.name)
+            ) {
+              const replaceToken =
+                node.property.type === "Identifier"
+                  ? node.property.name
+                  : node.property.type === "Literal"
+                  ? (node.property.value as string)
+                  : null;
```
Comment (wise-king-sullyman) [inline packages/eslint-plugin-pf-codemods/src/rules/v6/componentGroups-logSnippetRenameLeftBorderVariant/componentGroups-logSnippet-rename-leftBorderVariant.ts:68]: Super small nit, but IMO a quick switch here would be a lot cleaner than a ternary chain

**Pull request merged**